### ISSUE TITLE
Tetra tools lose tint.

### ISSUE THREAD

**Issue by TobiasX128** [Asek3/Rubidium#323]
when using tetra and rubidium, when using a fuller or wrapping handle, most other parts of the tool lose tint, reverting to white.

**Comment by RedPxnda** [comment]
Same thing applies for me:
Rubidium Version 5.4.0
Tetra Version 4.10.1


Warforging also gets miscolored, as well as the whole head if both the handle is wrapped and the item is warforged, the head loses all of its tint.

Would anyone be able to have a look in Rubidium's code to see what is causing this?

**Comment by pipocacomatum** [comment]
yeah, im also having the same issues with rubidium 0.5.4 and tetra 4.10.1

**Comment by Rikka-Lyn** [comment]
Add 'mixin.features.item=false' into the rubidium config can fix this issue when playing.
reference:
https://mineyourmind.net/forum/threads/how-to-fix-broken-t-construct-item-textures.32974/
